SPECS/abseil-cpp/abseil-cpp.spec

```diff
@@ -11,12 +14,19 @@ Source0:        https://github.com/abseil/abseil-cpp/archive/refs/tags/%{version
 BuildRequires:  cmake >= 3.20.0
 BuildRequires:  gcc
 BuildRequires:  make
-
-%if 0%{?with_check}
-BuildRequires:  gmock >= 1.12.0
-BuildRequires:  gmock-devel >= 1.12.0
-BuildRequires:  gtest >= 1.12.0
-BuildRequires:  gtest-devel >= 1.12.0
+BuildRequires:  ninja-build
+BuildRequires:  gcc-c++
+
+BuildRequires:  gmock
```

Comment:
Aren't these only required for tests? can you wrap them with the with_check macro?